docs/user-guide/configuring.md

```diff
@@ -420,6 +420,19 @@ If a rule has additional options, you can specify them using array literal synta
 
 This comment specifies the "double" option for the [`quotes`](../rules/quotes) rule. The first item in the array is always the rule severity (number or string).
 
+If the part preceded by `--` exists in the configuration comment, ESLint ignores the part. The `--` can be longer than 2. For examples:
+
+```js
+/* eslint eqeqeq: "off", curly: "error" -- You can clarify the reason of this configuration. */
+```
+
+```js
+/* eslint eqeqeq: "off", curly: "error"
+    --------
+    You can clarify the reason of this configuration.
+    But mind that '*' at beginning lines may cause syntax error. */
```

Comment:
I'm not sure I understand what this means. Is this an example message or a direction for the user reading the docs?

Reply:
I wanted to say that the following example doesn't work:

```js
/* eslint eqeqeq: "off", curly: "error"
 * --------
 * You can clarify the reason of this configuration.
 * But mind that '*' at beginning lines may cause syntax error.
 */
```

Reply:
Got it! I think it might be clearer to note just use this example directly. i.e.

```js
/* eslint eqeqeq: "off", curly: "error"
 * --------
 * This will not work due to the line above starting with a '*' character.
 */
```